src/components/PoliticianProfile/PoliticianProfile.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+	import type { Party } from '$models/party';
+
+	export let id: number;
+	export let firstname: string;
+	export let lastname: string;
+	export let avatar: string;
+	export let party: Party;
+	export let role: string;
+	export let isLarge = false;
+
+	$: fullname = `${firstname} ${lastname}`;
+</script>
+
+{#if isLarge}
+	<div class="p-2 font-sans">
+		<div class="flex gap-4">
+			<div class="w-[64px] h-[64px] relative">
+				<img
+					src={avatar}
+					alt={`${fullname}-${id}`}
+					class="w-full h-full rounded-full object-cover"
+					width="64"
+					height="64"
+					loading="lazy"
+					decoding="async"
+				/>
+				<div
+					class="w-[16px] h-[16px] absolute bottom-0 right-0 rounded-full border-solid border border-[#C0BFBE] bg-white overflow-hidden p-[2px] flex items-center justify-center"
+				>
+					<img
+						class="w-full h-full object-contain"
+						src={party.logo}
+						alt={party.name}
+						width="16"
+						height="16"
+						loading="lazy"
+						decoding="async"
+					/>
+				</div>
+			</div>
+			<div>
+				<p class="text-lg leading-[28px] text-text-01 font-bold font-serif">{fullname}</p>
+				<p class="leading-[18px] text-text-02">พรรค{party.name}</p>
+				<p class="leading-[18px] text-text-02">{role}</p>
+			</div>
+		</div>
+	</div>
+{:else}
```

Comment:
Small and large sizes should have the same HTML structure, can we refactor it to use the same HTML template but a different "style" or "property" instead? That way, we will have less duplicated code between small and large sizes.

Reply:
Fixed krub 🙏